storage/src/vespa/storage/persistence/filestorage/filestormanager.h

```diff
@@ -45,65 +45,43 @@ class FileStorManager : public StorageLinkQueued,
                         public framework::HtmlStatusReporter,
                         public StateListener,
                         private config::IFetcherCallback<vespa::config::content::StorFilestorConfig>,
-                        private MessageSender
+                        public MessageSender
```

Comment:
Just curious why this was changed from `private` to `public`

Reply:
If not it not possible to construct it with std::make_unique<>, as the MessageSender interface provided by *this would be private.